src/compas_blender/scene/meshobject.py

```diff
@@ -120,7 +140,7 @@ def draw_vertices(self) -> List[bpy.types.Object]:
         for vertex in vertices:
             name = f"{self.mesh.name}.vertex.{vertex}"
             color = self.vertexcolor[vertex]
-            point = self.vertex_xyz[vertex]
+            point = self.mesh.vertex_coordinates(vertex)
```

Comment:
I did this here because the `update_object` will apply transformation afterwards. If using vertex_xyz, the transformation will be applied twice, similar changes are done for both `BlenderMeshObject` and `BlenderVolmeshObjectr`